qdrant-landing/content/documentation/private-cloud/qdrant-cluster-management.md

```diff
@@ -195,3 +195,92 @@ spec:
           name: qdrant-tls
           key: tls.key
 ```
+
+### Configuring TLS for Inter-cluster Communication
+
+*Available as of Operator v2.2.0*
+
+If you want to encrypt communication between Qdrant nodes, you need to enable TLS by providing
+certificate, key, and root CA certificate used for generating the former.
+
+Similar to the instruction stated in the previous section, you need to create a secret:
+
+```shell
+ kubectl create secret generic qdrant-p2p-tls \
+        --from-file=tls.crt=qdrant-nodes.crt \
+        --from-file=tls.key=qdrant-nodes.key \
+        --from-file=ca.crt=root-ca.crt
+        --namespace the-qdrant-namespace
+```
+
+The resulting secret will look like this:
+
+```yaml
+apiVersion: v1
+data:
+  tls.crt: ...
+  tls.key: ...
+  ca.crt: ...
+kind: Secret
+metadata:
+  name: qdrant-p2p-tls
+  namespace: the-qdrant-namespace
+type: Opaque
+```
+You can reference the secret in the QdrantCluster spec:
+
+```yaml
+apiVersion: qdrant.io/v1
+kind: QdrantCluster
+metadata:
+  name: test-cluster
+  labels:
+    cluster-id: "my-cluster"
+    customer-id: "acme-industries"
+spec:
+  id: "my-cluster"
+  version: "v1.13.3"
+  size: 2
+  resources:
+    cpu: 100m
+    memory: "1Gi"
+    storage: "2Gi"
+  config:
```

Comment:
I think enabling p2p tls is missing here.

Reply:
nope, it gets automatically enabled when `caCert` is specified.